pulp_plugin_template/app/models.py

```diff
@@ -0,0 +1,55 @@
+from gettext import gettext as _
+from logging import getLogger
+
+from pulpcore.plugin.models import (Artifact, Content, ContentArtifact, RemoteArtifact, Importer,
+                                    ProgressBar, Publisher, RepositoryContent, PublishedArtifact,
+                                    PublishedMetadata)
+from pulpcore.plugin.tasking import Task
+
+log = getLogger(__name__)
+
+
+class PluginTemplateContent(Content):
+    """
+    The "plugin-template" content type.
+    """
+    TYPE = 'plugin-template'
+
+    # path = models.TextField(blank=False, null=False)
+    # digest = models.TextField(blank=False, null=False)
+    # objects = 
```

Comment:
I think a comment like:
```
# DEFINE FIELDS HERE
```
instead of commented out fields definitions would be clearer.